lib/src/follower.dart

```diff
@@ -241,8 +240,11 @@ class RenderFollowerLayer extends RenderProxyBox {
 
   @override
   bool hitTestChildren(BoxHitTestResult result, {required Offset position}) {
+    final followerOffset = _previousFollowerOffset ?? Offset.zero;
+    final transform = layer?.getLastTransform()?..translate(followerOffset.dx, followerOffset.dy);
```

Comment:
I see that we're accessing the "transform" of layer. Can that transform include a rotation? If so, it would seem that we need to figure out how the `followeOffset` maps to that transformed space.

Reply:
The transform could include a rotation, but it doesn't seem we are applying rotation at any place.

Reply:
Ok. We can leave this as-is for now. We might run into issues when widgets are rotated. We'll see.